pages.zh/common/gpg-zip.md

```diff
@@ -1,7 +1,7 @@
 # gpg-zip
 
 > 使用`GPG`加密存档中的文件和目录。
-> 更多信息：<https://www.gnupg.org/documentation/manuals/gnupg/gpg_002dzip.html>。
+> 更多信息： <https://www.gnupg.org/documentation/manuals/gnupg/gpg_002dzip.html>.
```

Comment:
I think this is the correct period:
```suggestion
> 更多信息： <https://www.gnupg.org/documentation/manuals/gnupg/gpg_002dzip.html>。
```

Reply:
It is, but #5240 isn't quite done yet, so maybe we should wait on this one. 🤔